modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/configuration/CPDefinitionLinkTypeSettingsImpl.java

```diff
@@ -29,22 +39,34 @@ public String[] getTypes() {
 	}
 
 	@Activate
-	protected void activate(BundleContext bundleContext) {
+	protected void activate(
+		BundleContext bundleContext, Map<String, Object> properties) {
```

Comment:
you just need a `CPDefinitionLinkTypeConfiguration` reference. Please get rid of the `_serviceTrackerMap`. 

Thanks